app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less

```diff
@@ -464,4 +464,217 @@
+            width: 50%;
+            margin: 0;
+            input {
+                display: inline-block;
+                width: 100%;
+                background: transparent;
+                border: 1px solid @color-maksym__main-gray;
+                height: 49px;
+                line-height: 49px;
+                color: @color-maksym__main-gray;
+                .lib-font-size(14);
+                padding-left: 20px;
+                padding: 0 0 0 20px;
+                margin: 0;
+            }
+            button {
+                padding: 17px 35px;
+                border: 0;
+                border-radius: 0;
+                margin: 0;
+                color: @color-maksym__white;
+                background: @color-maksym__main-black;
+                font-style: normal;
+                .lib-font-size(14);
+                font-weight: @font-weight__regular;
+                text-transform: uppercase;
+                letter-spacing: 1px;
+            }
+            .block.newsletter .action.subscribe {
+                border-radius: 0;
+            }
+        }
+        .block.newsletter {
+            .field {
+                .control:before {
+                    display: none;
+                }
+            }
+        }
+        .block.newsletter {
+            .form.subscribe {
+                display: flex;
+                align-items: center;
+                width: 100%;
+            }
+        }
+        .block.newsletter {
+            .field.newsletter {
+                max-width: 46%;
+            }
+        }
+    }
+
+    /* FOOTER */
+    div.footer.content {
+        max-width: 100%;
+        margin: 0;
+        border: 0;
+    }
+    .footer {
+        h3 {
+            margin-bottom: 40px;
+            .lib-font-size(16);
+            font-weight: @font-weight__heavier;
+            line-height: 16px;
+            text-transform: uppercase;
+        }
+
+        .footer__links-thumb__sub-list__item {
+            display: flex;
+            align-items: center;
+            position: relative;
+            padding-bottom: 15px;
+            height: 22px;
+            &:hover,
+            &:focus {
+                &::after {
+                    opacity: 1;
+                }
+                .footer__links-thumb__sub-list__link {
+                    left: 15px;
+                    color: @color-maksym__main-black;
+                }
+            }
+            &::after {
+                display: block;
+                opacity: 0;
+                content: " ";
+                width: 7px;
+                height: 7px;
+                margin-right: 10px;
+                border-radius: 50%;
+                background-color: @color-maksym__main-black;
+                transition: all 0.4s ease 0s;
+            }
+        }
+        .footer__links-thumb__sub-list__link {
+            position: absolute;
+            left: 0;
+            display: flex;
+            align-items: center;
```

Comment:
Please use mixin for `display: flex` and `align-items: center`

Reply:
done